war/src/main/js/util/jenkins.js

```diff
@@ -184,12 +184,36 @@ exports.loadTranslations = function(bundleName, handler, onError) {
 			}
 			throw 'Unable to load localization data: ' + res.message;
 		}
+		
+		var translations = res.data;
+		
+		if('undefined' !== typeof(Proxy)) {
+			translations = new Proxy(translations, {
+				get: function(target, property, receiver) {
+					if(property in target) {
+						return target[property];
+					}
+					console.log('"' + property + '" not found in translation bundle.');
+					return property;
+				}
+			});
+		}
 
-		handler(res.data);
+		handler(translations);
 	});
 };
 
 /**
+ * Load a handlebars template from the server
+ */
+exports.loadTemplate = function(templateName, handler, onError) {
```

Comment:
Maybe an 🐜  but I have to wonder if there isn't a better way to do this... perhaps by doing a little maven configuration to throw our JS/templates/etc resources in WEB-INF so they can be directly exposed from the WAR as simple URLs?  Feels like we're reinventing the wheel by writing a custom function to serve static content using an URL param vs. using parts of the Servlet spec.